app/models/amp/Amp.scala

```diff
@@ -26,7 +26,7 @@ import typeclasses.MongoKey
 import uk.gov.hmrc.http.cache.client.CacheMap
 import uk.gov.hmrc.play.config.ServicesConfig
 
-final case class Amp(id: String,
+final case class Amp(_id: String,
```

Comment:
Might be worth naming this credId for clarity?

Reply:
I wanted to keep this exactly as it is stored in AMP mongo for the time being in case we wanted to switch back. 

We may have a discussion and do a refactoring exercise to remove these additional fields however; that could be its own ticket if we decide we want to go that way.